apps/test/integration/levelSolutions/applab/ec_canvas.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
         'rect(80, 120, 160, 240);\n' +
         'setStrokeWidth(3);\n' +
         'setStrokeColor("red");\n' +
-        'setFillColor("yellow");\n' +
+        'setFillColor(rgb(255,0,0));\n' +
```

Comment:
not totally sure this is the right thing to do here for testing, but I couldn't find anything else for these blocks.